Tasks/VsTestV2/task.loc.json

```diff
@@ -69,7 +69,7 @@
       "name": "testAssemblyVer2",
       "type": "multiLine",
       "label": "ms-resource:loc.input.label.testAssemblyVer2",
-      "defaultValue": "**\\*test*.dll\n!**\\*TestAdapter.dll\n!**\\obj\\**",
+      "defaultValue": "**\\bin\\**\\test.dll\n**\\bin\\**\\tests.dll\n",
```

Comment:
It should be *test.dll and *tests.dll, we can copy from task.json file above. That looks good

Reply:
The new filter says
```suggestion
      "defaultValue": "**\\bin\\**\\test.dll\n**\\bin\\**\\tests.dll",
```

Reply:
![image](https://github.com/microsoft/azure-pipelines-tasks/assets/92331194/983eb7a5-e036-45c2-8cc3-841834cb85bd)
Is this difference expected?

Reply:
It is not, you are right, I am missing stars there, fixing.